TwitchPlaysAssembly/Src/Votes.cs

```diff
@@ -19,6 +20,33 @@ public class VoteData
 	internal Action onSuccess;
 }
 
+public class VotesolveData
```

Comment:
Implementing another class just for vote solves has made so you have to keep handling vote solves separately. Could you make this class just extend VoteData so that we don't do that?

validityChecks could be used for the Detonation vote type, so that could be put into the base class.

Reply:
The reason I can't do that is because the "PossibleVotes" dictionary is static which means that I can't access the module that needs to be solved inside of it.